roles/openshift_aws/tasks/wait_for_node.yml

```
@@ -0,0 +1,18 @@
+---
+# This will wait for the node to become ready status
+- name: wait for node become ready
+  oc_obj:
+    name: "{{ node_name }}"
+    kind: node
+    state: list
+  delegate_to: "{{ groups.masters.0 }}"
+  register: get_node
+  until:
+    - get_node.module_results.results[0] is defined
+    - get_node.module_results.results[0].status is defined
+    - get_node.module_results.results[0].status.conditions is defined
+    - get_node.module_results.results[0].status.conditions | selectattr('type', 'match', '^Ready$') | map(attribute='status') | join | bool == True
+  retries: 150
+  delay: 5
```

```suggestion
  delay: 10
```
10*150 = 7500 seconds = 25 minutes, should be sufficient. A bit increased delay would prevent from spamming API server with requests

---

make sense